src/webgpu/api/validation/error_scope.spec.ts

```diff
@@ -146,7 +155,7 @@ Tests that popping an empty error scope stack should reject.
   )
   .fn(t => {
     const promise = t.device.popErrorScope();
-    t.shouldReject('OperationError', promise);
```

Comment:
this is fine since we already test popErrorScope has stacks elsewhere, but why is this needed?

Reply:
I'm actually not entirely sure? Maybe I had an outdated version of chromium that didn't plumb the stack through, but when I was running the tests, without this they were all failing with the missing stack error... I can't easily rebase/update my chromium atm so I just added these suppressions for the time being. If we are confident it should work without them I can remove them also.

Reply:
Actually, since I'm OOO I will just land this for now. If it turns out we don't need the allow `allowMissingStacks` we can fix them in a follow up!

Reply:
You didn't need to check back on this while OOO! I was assuming it was just an old chromium build, and I was going to try removing these changes, but they really don't matter, so no need to follow up.